src/v2/System/i18n/i18n.ts

```diff
@@ -0,0 +1,78 @@
+import i18n from "i18next"
+import { initReactI18next } from "react-i18next"
+import LanguageDetector from "i18next-browser-languagedetector"
+
+const locales = ["en-US", "de-DE"]
+
+i18n
+  .use(initReactI18next)
+  .use(LanguageDetector)
+  .init({
+    debug: true,
+    // backend: {
+    //   loadPath: "assets/locales/{{lng}}/{{ns}}.json",
+    // },
```

Comment:
Ideally, we would like the JSON files to be loaded like this, but we couldn't make it work (serving the static assets using webpacker... any ideas?).

Reply:
we tried a few options but none worked, basically, we need to serve the locale JSON files somewhere, we tried under /assets but got lost in the setup of how to do it, any suggestions? cc @damassi 😬

Reply:
Happy to pair on this when y'all have a moment to pick this back up!

Reply:
Thanks @damassi, that would be great! When would be a good moment for you?

Also: is there something that should be prepared beforehand?

Reply:
I'm back from OOO; please reach out in slack!